e2e/test_operator_backups/operator_backup_test.go

```diff
@@ -68,6 +68,8 @@ var _ = BeforeSuite(func() {
 
 	// Create a blobstore for testing backups and restore
 	factory.CreateBlobstoreIfAbsent(fdbCluster.Namespace())
+
+	// Note: Encryption key secret is automatically created during namespace setup
```

Comment:
Is there anything we should change in the e2e test case to ensure we test backups with and without encryption (and also validate that the backup is actually encrypted)?

Reply:
That's a great point. 
I'm trying to add a file - metadata that contains if backup is encrypted or not.
After that we can check that metadata file has encryption enabled in the test and 
if encryption is enabled and restore works then that can give us a some kind of gurantee.